### PR TITLE
Microverse Portfolio 

### PR DESCRIPTION
**Pull request by shahira-sadat**
This is first part of Microverse portfolio.
Navigation bar and header sections added.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header class="main-section">
+
+        <section class="toolbar">
+          <nav class="desktop">
+            <div class="all-item"><h1><a href="#">My Logo</a></h1></div>
```
Comment (danmainah) [inline index.html:17]: - [ ] Kindly remove the logo from the `nav` tag, only the hamburger menu is supposed to have a `nav` tag.

Comment (danmainah) [reply index.html:17]: - [ ] Kindly check the alignment of your header section elements,make sure they align as in the design given

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        </section>
+      </header>
+
+    <main class="main-parts">
+        <section class="headline-section">
+          <h1>I'm Shahira <br> Glad to see <br>you!</h1>
+          <p>I’m a software developer! I can help you build a product , feature or website Look through<br> some of my
+            work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+            contact me.</p>
+          <P class="connect">LET'S CONNECT</P>
```
Comment (danmainah) [inline index.html:34]: - [ ] Kindy make sure your `LET'S CONNECT` has an `<a> or a `<button>` tag.This is because let's connect is a link.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @shahira-sadat , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR and Description


# Required Changes ♻️
- [ ] Kindly add the get started section and a description to your readme file, making it more professional.
_Check the comments under the review._

## Optional suggestions



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        <div class="all-item"><a href="#">My Logo</a></div>
+        <div class="desk-item">
+          <div><a href="#portfolio">Portfolio</a></div>
+          <div><a href="#about">About</a></div>
+          <div><a href="#contact">Contact</a></div>
+        </div>
+        <div class="item menu"><a href="#" class="openMenu"><i class="fa fa-bars" aria-hidden="true"></i></a></div>
+      </nav>
+    </section>
+  </header>
```
Comment (Mre55) [inline index.html:27]: As requested by the previous reviewer the `logo` should have to be out of the `nav` tag and inside the `header` tag. Here I put the general structure of how content inside the `header`  tag should have to be 👇 

```
<header>
    <a>Log...</a>
    <nav>...</nav>
</header>

```

**Review by Mre55 — CHANGES_REQUESTED** [review]
Hi @shahira-sadat 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 

 ## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions 🟡 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**